src/EnergyPlus/Data/CommonIncludes.hh

```diff
@@ -183,7 +183,6 @@
 #include <EnergyPlus/HeatBalanceHAMTManager.hh>
 #include <EnergyPlus/HeatBalanceIntRadExchange.hh>
 #include <EnergyPlus/HeatBalanceInternalHeatGains.hh>
-#include <EnergyPlus/HeatBalanceKivaManager.hh>
```

Comment:
The Kiva HB Manager header did not have any state data, so that was just nicely cleaned out of here.